services/QuillLMS/client/app/bundles/PremiumHub/components/dataExportTableAndFields.tsx

```diff
@@ -242,30 +257,46 @@ export const DataExportTableAndFields = ({ queryKey, searchCount, selectedGrades
   }
 
   return(
-    <div className="data-export-container">
-      <section className="fields-section">
-        <h3>Fields</h3>
-        <div className="fields-container">
-          {renderCheckboxes()}
-        </div>
-      </section>
-      <section className="preview-section">
-        <h3>Preview</h3>
-        <div className="preview-disclaimer-container">
-          <img alt={informationIcon.alt} src={informationIcon.src} />
-          <p>This preview is limited to the first 10 results. Your download will include all activities.</p>
-        </div>
-      </section>
-      {loading && <Spinner />}
-      {!loading && <DataTable
-        className="data-export-table reporting-format"
-        defaultSortAttribute="completed_at"
-        defaultSortDirection="desc"
-        emptyStateMessage={noResultsMessage('activity')}
-        headers={getHeaders()}
-        rows={data || []}
-      />}
-    </div>
+    <React.Fragment>
```

Comment:
General note: there was a style regression this week in `develop`, such that the button is right-shifted with respect to this component.